whitelabelco/sofo-foods
# Cp/feat/history copy

### Description 💻

Resolves: #213 

### Implementation 💡

updates copy on History page


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- scroll down and click "Our History"
- confirm content updates match the tickets

### Screenshots/Demos 📷

### Checklist ✅
